pkg/kubectl/cmd/get/get_test.go

```diff
@@ -631,6 +631,31 @@ func TestGetObjectIgnoreNotFound(t *testing.T) {
 	}
 }
 
+func TestEmptyResult(t *testing.T) {
+	cmdtesting.InitTestErrorHandler(t)
+
+	tf := cmdtesting.NewTestFactory().WithNamespace("test")
+	defer tf.Cleanup()
+	codec := scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
+
+	tf.UnstructuredClient = &fake.RESTClient{
+		NegotiatedSerializer: resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer,
+		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
+			return &http.Response{StatusCode: 200, Header: cmdtesting.DefaultHeader(), Body: cmdtesting.ObjBody(codec, &corev1.PodList{})}, nil
+		}),
+	}
+
+	streams, _, _, errbuf := genericclioptions.NewTestIOStreams()
+	cmd := NewCmdGet("kubectl", tf, streams)
+	// we're assuming that an empty file is being passed from stdin
+	cmd.Flags().Set("f", "-")
+	cmd.Run(cmd, []string{"pods"})
```

Comment:
this is the equivalent of running `kubectl get -f - pods`? is that valid?

Reply:
correct, b/c you can specify multiple resource to get them, but in this case unnecessary, I'll drop `pods` from there.

Reply:
TIL, when setting flags (`cmd.Flags().Set(...)`) you need to use flag's full name, not the shorthand since the latter will not get resolved.